alphamountain/package.yaml

```diff
@@ -114,21 +116,23 @@ pipelines:
       // tql2
       subscribe "alphamountain"
       where @name == "alphamountain.category"
-      legacy "context update alphamountain-categories --key=hostname --create-timeout {{ inputs.threat-expiry }}"
+      context::update "alphamountain-categories", key=hostname, create_timeout={{ inputs.threat-expiry }}
 
 examples:
   - name: Match all string and IP fields
     description: |
       Starts a lookup that matches all historic and newly arriving events
       against the `alphamountain-threats` context.
     definition: |
-      lookup alphamountain-threats --field :string,:ip
+      // tql2
+      legacy "lookup alphamountain-threats --field :string,:ip"
```

Comment:
Let's remove this `lookup` example, as there are simpler ways to illustrate what's going on.